google/resource_compute_region_instance_group_manager.go

```diff
@@ -145,6 +148,14 @@ func resourceComputeRegionInstanceGroupManager() *schema.Resource {
 					},
 				},
 			},
+
+			"distribution_policy": &schema.Schema{
```

Comment:
is there a shared zone names validator we could use here?

Reply:
I don't believe so?  Do you have an example in mind of what the validator would be doing?

Reply:
search for "ValidateFunc", they have some internal functional for basic validation:

https://github.com/terraform-providers/terraform-provider-google/blob/fec2f0bc02d7a4a19fe18bdcdd573f255d7ce83e/google/validation.go

However, it doesn't look like they use any of "Live" (aka self-maintaining, pulled from API) data to perform validations of input. So, I guess it's fine